lib/ripper-tags.rb

```diff
@@ -35,17 +35,24 @@ def self.default_options
   class ForgivingOptionParser < OptionParser
     attr_accessor :ignore_unsupported_options
 
-    def parse(argv)
-      argv = argv.dup
+    def load_options_file(file)
+      @argv.unshift(*File.read(file).split(/\r?\n/).flat_map { |line|
```

Comment:
I was thinking that `IO.readlines` does exactly this (upto `flat_map`), but only with the platform-relevant separator (if we want to support old rubies - line separator can be passed as an arg since 1.9.2).